gradle/mixins/checkstyle.gradle

```diff
@@ -0,0 +1,9 @@
+// Defines shared Gradle project Checkstyle analyzer configuration.
+
+apply plugin: 'checkstyle'
+
+checkstyle {
+    toolVersion '8.21'
```

Comment:
Guessing it is better practice to set the version here instead of `gradle.properties` because it defines the tool version and not the plugin version?  I'm just looking to learn, no issues whatsoever with this.

Reply:
Ha, good catch. Tbh, I didn't really think about it. Now that you pointed that out, `gradle.properties` seems a better place for consistency with the other version information. Moved. Thank you :pray:.